pkg/ruler/remotequerier.go

```diff
@@ -209,7 +222,9 @@ func (q *RemoteQuerier) Read(ctx context.Context, query *prompb.Query) (*prompb.
 	if len(rdResp.Results) != 1 {
 		return nil, errors.Errorf("responses: want %d, got %d", 1, len(rdResp.Results))
 	}
-	return rdResp.Results[0], nil
+
+	res := rdResp.Results[0]
+	return remote.FromQueryResult(sortSeries, res), nil
```

Comment:
I don't understand why `sortSeries` is a parameter now. Did the remote-read API also change? Previously they were supposed to be returned sorted by the server. Are we supposed to pass `sortSeries` to the server?

Reply:
strange